javascript/javascript3/week1/weather.js

```diff
@@ -0,0 +1,180 @@
+//COOL API
+
+/*As a Pokemon master, I chose the a API that has information
+about Pokemons. When I call this specific API, I get an
+object with string containing the options of 
+endpoints depending on what information you want 
+(generation, berries, moves, etc.). For example, if I choose 
+the "generation" endpoint, I will get an object with 
+information like the total generations of Pokemon (so far), 
+an array with objects of the name and endpoint of each
+specific generation.*/
+fetch('https://pokeapi.co/api/v2/')
+    .then(response => response.json())
+    .then(pokemonData => {
+        console.log(pokemonData);
+        let generationURL = pokemonData.generation
+        fetch(generationURL)
+            .then(response => response.json())
+            .then(pokemonData => {
+                console.log(pokemonData);
+            });
+
+    });
+
+//WEATHER API
+
+//Global variables
+let inputLocation = document.getElementById("location");
+let cityButton = document.getElementById("submit-city");
+let locationButton = document.getElementById("user-location");
+let clearButton = document.getElementById("clear-history")
```

Comment:
Instead of let here you can use const, as you are not going to rewrite those variables.